docs/guidelines/accessibility.md

```diff
@@ -1,3 +1,659 @@
+
+As technology evolves, some methods to meet the guideline could get outdated. We will update outdated ones though our periodical accessibility testing.
+
+We value web accessibility, and consider it is important to promote it to anybody who own and work on websites. Here, you'll find resources for web accessibility from basic concepts to technical approaches.
+
+[TO DO:  ADD SOME QUICK LINKS TO RESOURCES.]
+
+
+## What is web accessibility?
+
+Web accessibility means **making your website available to as many people as possible**.
+
+It’s a combination of common sense, empathy and understanding. 
+
+Source: [WUHCAG](https://www.wuhcag.com/what-is-web-accessibility/)
+
+Web accessibility refers to the **inclusive practice of removing barriers that prevent interaction with, or access to websites, by people with disabilities**.
+
+Source: [Wikipedia](https://en.wikipedia.org/wiki/Web_accessibility)
+
+Web accessibility consists of two parts:
+
+- philosophy &mdash; equivalent user experience
+- process &mdash; techniques + code (what we build on line)
+
+They both are equally important to achieve the goal of creating an accessible website.
+
+## Why web accessibility matters
+
+At its core, the Web is about people being able to find and access information. Increasingly, access to online information is essential to our daily lives.  Benefits from the Web are supposed to be avaiable to anybody.  There shouldn't be any exception to any of us. 
+
+When it’s broken down, the word accessibility is the “ability” to “access” and really this is why the topic is so important. Broadly stated, it’s intentionally designing the world to include everyone, regardless of disability. So we have access to more of life. 
+
+For people with disabilities who have access to the Internet, independent access to information offers an unprecedented amount of freedom. 
+
+According to [WebAIM](https://webaim.org/), most studies find that about one fifth (20%) of the population has some kind of disability.  
+
+Even though not all of these people have disabilities that make them difficult to access the internet, it is still a significant portion of the population. This number would increase as senior population grows as projected by [United Nations Population Fund (UNFPA)](https://www.unfpa.org/publications/ageing-twenty-first-century). As they age, people could have these problems in various levels and some would have more than one problem. Remember, we are all temporarily able-bodied.
+
+
+Resources:
+
+- [Brailworks: Why Accessibility Matters](https://brailleworks.com/accessibility-matters/)
+- [Stanford University Online Accessibility Program: Why Accessibility Matters](https://soap.stanford.edu/getting-started/why-accessibility-matters)
+
+
+## Myths of web accessibility
```

Comment:
i like the myths a lot! just wondering if it's already stated in the `Bureau of Internet Accessibility: Website Accessibility Isn't What You Think It Is: 10 Myths, Busted`, do we want to repeat it or directly point people to it. A content team question too